BUILDING.md

```diff
@@ -2,8 +2,9 @@
 
 UEFI applications are simple COFF (Windows) executables, with the special
 `EFI_Application` subsystem, and some limitations (such as no dynamic linking).
-[Rust supports building UEFI applications](https://github.com/rust-lang/rust/pull/56769)
-though the `x86_64-unknown-uefi` target.
+Rust supports building UEFI applications though the
```

Comment:
I've realized there's a typo here, it should be `building UEFI applications for the ... targets` :sweat_smile: could you update the text?

Reply:
fixed